client/src/pages/club/match/EditMatch.tsx

```diff
@@ -1,9 +1,365 @@
+    console.log(datas)
+    setMatchDatas(datas);
+  }
+
+  const dateChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setDate(e.target.value);
+  };
+
+  const timeChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setTime(e.target.value);
+  };
+
+  const mapSettingModalHandler = () => {
+    setIsOpenMapSetting(!isOpenMapSetting);
+  };
+
+  const mapViewModalHandler = () => {
+    if (!placeValue) return;
+    setIsOpenMapView(!isOpenMapView);
+  };
+
+  const openAddMemberHandler = () => {
+    setIsOpenAddMember(!isOpenAddMember);
+  };
+  
+  const validationCheck = () => {
+    if(!placeValue || !time ) {
+      alert('*가 표시된 항목은 필수 입력란입니다.');
+      return;
+    }
+  }
+
+  if (!candidateList.length && isOpenAddMember) {
+    setIsOpenAddMember(false);
+  }
+
+  useEffect(() => {
+    setRequestDatas(date, time, placeValue, candidates, teamList, records);
+  },[records])
+
+  return (
+    <S_Container>
+      <S_Title>경기 수정</S_Title>
+      <div style={{ marginTop: '15px', marginBottom: '15px' }}>
+        <S_Label>날짜/시간 선택 *</S_Label>
+        <S_Input type='date' value={date} onChange={dateChangeHandler} />
+        <S_Input type='time' value={time} onChange={timeChangeHandler} />
+      </div>
+      <div style={{ marginTop: '15px', marginBottom: '15px' }}>
+        <S_Label>장소 *</S_Label>
+        <S_Input type='text' value={placeValue?.place_name} readOnly />
+        <S_SelectButton onClick={mapSettingModalHandler} style={{width: "auto"}}>지도설정</S_SelectButton>
+        <S_SelectButton onClick={mapViewModalHandler} style={{width: "auto"}}>지도보기</S_SelectButton>
+        {isOpenMapSetting && (
+          <S_MapBackdrop onClick={mapSettingModalHandler}>
+            <S_MapView onClick={(e) => e.stopPropagation()}>
+              <KakaoMapSearch
+                mapSettingModalHandler={mapSettingModalHandler}
+                setPlaceValue={setPlaceValue}
+              />
+            </S_MapView>
+          </S_MapBackdrop>
+        )}
+        {isOpenMapView && (
+          <S_MapBackdrop onClick={mapViewModalHandler}>
+            <S_MapView onClick={(e) => e.stopPropagation()}>
+              {/* <KakaoMapView place={placeValue} /> */}
+              {
+                placeValue &&
+                <KakaoMapView y={placeValue.y} x={placeValue.x}/>
+              }
+            </S_MapView>
+          </S_MapBackdrop>
+        )}
+      </div>
+      <div style={{ marginTop: '15px', marginBottom: '15px' }}>
+        <S_Label>참석자</S_Label>
+        <S_Description>
+          경기를 등록하면 경기정보 페이지에서 참석/불참을 선택할 수 있어요.
+        </S_Description>
+        <S_Description>
+          참석을 선택한 멤버는 자동으로 등록됩니다. 
+          <S_EditButton style={{padding:"0 7px",float:"right"}}>추가</S_EditButton>
+        </S_Description>
+        
         <div>
-            경기 수정 페이지
+        {candidates &&
+          candidates.map((member, idx) => {
+            return <S_NameTag key={idx}>{member}</S_NameTag>;
+          })}
```

Comment:
아래처럼도 사용 가능합니다. 참고만해주세요~
```suggestion
        {candidates?.map((member, idx) => {
            return <S_NameTag key={idx}>{member}</S_NameTag>;
          })}
```

Reply:
감사합니다!